### PR TITLE
Feature/127325 submit ticket sidebar

### PR DESCRIPTION
**Pull request by bryan-farris**
Adds JS & CSS to add sidebar on the Submit Ticket page. 

🎫 https://central.tri.be/issues/127325

### CODE REVIEW

File: js/includes/submitTicket.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * 
+ */
+var $submitTicketForm = $( '#SubmitTicketForm' ).parent();
+var submitTicketSidebar = `
+	<div class="submit-ticket__sidebar col-md-4">
+		<h3>Psst... Want better support?</h3>
+		<p>Get the most out of your support request by adding more details to the ticket.
+		<ul>
+			<li>Are all of your plugins updated to the <a href="https://tec.tribe/category/products/release-notes/" target="_blank" title="The Events Calendar Release Notes">Latest Version</a>?</li>
+			<li>Copy and paste your <a href="https://support.theeventscalendar.com/244018-Sharing-Your-System-Information" target="_blank" title="Sharing your system information">System Information</a></li>
```
Comment (ryanurban) [inline js/includes/submitTicket.js]: Read this for these external links, quite interesting ; ) https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

File: js/includes/submitTicket.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+/**
+ * Update Bootstrap classes on Submit Ticket Form to make room for sidebar
+ * 
+ */
+
+if ( isLoggedIn && isSubmitTicketPage === true ) {
+	$( '.page-submit_ticket' ).find( '.col-lg-10.col-md-12.col-lg-offset-1' ).each( function() {
+		$( this ).removeClass( 'col-lg-10 col-md-12 col-lg-offset-1' ).addClass( 'submit-ticket__form col-md-8 ');
```
[the remark this answers is not in the document]
Comment (bryan-farris) [reply js/includes/submitTicket.js]: Good catch and good call! 👍

File: js/includes/submitTicket.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ * 
+ * Request to add some helpful notes for users to open better tickets. 
+ * Will hopefully reduce "back and forth" of the initial support process.
+ * 
+ */
+var $submitTicketForm = $( '#SubmitTicketForm' ).parent();
+var submitTicketSidebar = `
+	<div class="submit-ticket__sidebar col-md-4">
+		<h3>Psst... Want better support?</h3>
+		<p>Get the most out of your support request by adding more details to the ticket.
```
[the remark this answers is not in the document]
Comment (bryan-farris) [reply js/includes/submitTicket.js]: 😬 thanks!

File: js/includes/submitTicket.js (hunk 1 above)
Comment (bryan-farris) [reply js/includes/submitTicket.js]: 😮

**Pull request merged**